README.md

```diff
@@ -88,8 +88,11 @@ of the course in the `book/` directory. Note that you have to separately build
 and zip exercises and add them to `book/html`. To build any of the translated
 versions of the course, run `MDBOOK_BOOK__LANGUAGE=xx mdbook build -d book/xx`
 where `xx` is the ISO 639 language code (e.g. `da` for the Danish translation).
+
 [TRANSLATIONS.md](TRANSLATIONS.md) contains further instructions.
 
+For complete info about installing necessary tools , refer [CONTRIBUTING.md](CONTRIBUTING.md#formatting-and-preparation)
```

Comment:
Actually, let us take this out since this is just about how to view a translation:
```suggestion
[TRANSLATIONS.md](TRANSLATIONS.md) contains further instructions.
```